repose-aggregator/components/filters/herp/src/test/scala/org/openrepose/filters/herp/HerpFilterTest.scala

```diff
@@ -154,6 +213,19 @@ class HerpFilterTest extends FunSpec with BeforeAndAfterAll with BeforeAndAfter
       logEvents.size shouldBe 1
       logEvents.get(0).getMessage.getFormattedMessage should include("\"Method\" : \"POST\"")
     }
+    it("should expose the cadf method") {
+      // given:
+      servletRequest.setMethod("POST")
+
+      // when:
+      herpFilter.configurationUpdated(herpConfig)
+      herpFilter.doFilter(servletRequest, servletResponse, filterChain)
+
+      // then:
+      def logEvents = listAppenderPre.getEvents
+      logEvents.size shouldBe 1
+      logEvents.get(0).getMessage.getFormattedMessage should include("\"CadfMethod\" : \"update/post\"")
+    }
```

Comment:
They are testing two different things. One is testing the formatter, the other is testing to make sure it's exposed to a template. Both things that need to happen. They are testing the individual unit that i care about in each case. A unit test shouldn't be checking on two different things.